### DISCUSSION TITLE
problemas SCORM en versiones PHP

### DISCUSSION THREAD

**Discussion by chatonito** [chamilo/chamilo-lms#5287] [General] [answered]
en las lecciones SCORM se ve diferente con las versiones php
a).- en la version php7.4 las lecciones SCORM se ven correctamente en su edición

b).- en la version php8.2 Las lecciones SCORM se ven desconfiguradas cuando se intenta editarlas.

lo mismo ocurre cuando se edita las calificaciones y evaluaciones
![Captura de Pantalla 2024-03-22 a la(s) 11 42 58](https://github.com/chamilo/chamilo-lms/assets/164546688/e9504830-1c61-43dd-8886-3f4252995c47)
anexo
![Captura de Pantalla 2024-03-22 a la(s) 11 49 53](https://github.com/chamilo/chamilo-lms/assets/164546688/53aaa5d4-4075-4e30-b065-71ab57dcaed8)
 
anexo 2 pantallazos

¿como se puede corregir para que esas 2 partes: SCORM y Calificaciones se vean correctamente en chamilo 1.11.26
nuestro equipo es un Linux ubuntu 23.01

muchas gracias por su ayuda.
saludos
atte
dr. agustin rico guzman 
webmaster Enseñanza Médica CAR

**Comment by Kaneda-1** [comment, marked as answer]
Hola @chatonito
Ya se ha explicado otras veces el motivo. En Github y en el foro https://forum.chamilo.org/

Chamilo 1.x (1.11.26) solamente es compatible con PHP 7.4.x pero no lo es al 100% con la rama PHP 8.x. 

Por lo tanto no es compatible con PHP 8.0, 8.1, 8.2 y 8.3.

Aquí lo puedes comprobar:

https://github.com/chamilo/chamilo-lms/releases

> Although we do not officially support PHP 8.0, 8.1 or 8.2 with the 1.11 branch of Chamilo, we tested it relatively thoroughly with the -php74 package and it worked well, to the minor exception of the edition of learning paths which is visually a bit broken.
> 
> We recommend using PHP 7.4 while we are running to deliver Chamilo 2.0 with full support for PHP 8.2 or superior.

Un saludo.

**Reply by ossiof** [reply]
La versión de PHP es 7.4 que es la que se utiliza para Chamilo. Ya que no soporta versiones superiores.

**Comment by chatonito** [comment]
Muchas gracias por tu ayuda Kaneda y efectivamente chamilo 1.11.26 falla en esos 2 aspectos con php8.x SCORM, y solo en la edición y las evaluaciones y calificaciones, el resto se ve bien y hasta más rápido que con php7.4. solo que se me hizo curioso que solo esas 2 áreas de chamilo no funcionaran y pensé que se trataba de algo del codigo php en chamilo.
esperemos a una actualización, muchas gracias, saludos de morelia michoacán méxico
